MAiD - Case Management App/force-app/main/default/objects/Case_Contact__c/validationRules/Missing_preferred_contact_info.validationRule-meta.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<ValidationRule xmlns="http://soap.sforce.com/2006/04/metadata">
+    <fullName>Missing_preferred_contact_info</fullName>
+    <active>true</active>
+    <description>Error if contact info that matches preferred selection is not populated</description>
+    <errorConditionFormula>OR(
+	AND(ISPICKVAL(Preferred_method_of_contact__c, &apos;Phone&apos;),
```

Comment:
Simplification suggestion @dcstuff:
	AND(ISPICKVAL(Preferred_method_of_contact__c, &apos;Phone&apos;),
			ISBLANK(Contact_Person_Home_Phone_Number__c),
			ISBLANK(Contact_Person_Cell_Phone_Number__c),
			ISBLANK(Contact_Person_Work_Phone_Number__c))